src/app.js

```diff
@@ -1,21 +1,16 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const querystring = require('querystring');
 
-module.exports = sum;
+http.createServer((req, res) => {
+  const readUrl = new URL(`http://${req.headers.host + req.url}`);
+  const queryParams = querystring.parse(readUrl.search.slice(1));
+
+  const dataToSend = JSON.stringify({
+    parts: readUrl.pathname.slice(1).split('/'),
+    query: queryParams,
+  });
+
+  res.end(dataToSend);
```

Comment:
Before ending the response, you should set the Content-Type header to 'application/json' using res.setHeader method.